ui/imports/shared/views/TransactionDetailView.qml

```diff
@@ -212,23 +212,23 @@ Item {
                     id: baseFee
                     maxWidth: parent.width
                     primaryText: qsTr("Base fee")
-                    secondaryText: root.transaction !== undefined && !!root.transaction ?  qsTr("%1 Gwei").arg(RootStore.hex2Gwei(root.transaction.baseGasFees)) : ""
+                    secondaryText: root.transaction !== undefined && !!root.transaction ?  qsTr("%1").arg(LocaleUtils.currencyAmountToLocaleString(root.transaction.baseGasFees)) : ""
                 }
                 InformationTile {
                     maxWidth: parent.width
                     primaryText: qsTr("Tip")
-                    secondaryText: root.transaction !== undefined && !!root.transaction ?  qsTr("%1 Gwei <font color=\"#939BA1\">&#8226; Max: %2 Gwei</font>").
-                                                                    arg(RootStore.hex2Gwei(root.transaction.maxPriorityFeePerGas)).
-                                                                    arg(RootStore.hex2Gwei(root.transaction.maxFeePerGas)) : ""
+                    secondaryText: root.transaction !== undefined && !!root.transaction ?  qsTr("%1 <font color=\"#939BA1\">&#8226; Max: %2</font>").
```

Comment:
Shouldn't colours come from the palette?

Reply:
Yes!

Reply:
plus replace the color with something coming from Theme/Style of course :)

Reply:
please check the new style.
- Using `Theme.palette.baseColor1``
- `qsTr` now only wraps things that need to be translated